roboteam_ai/src/skills/Attack.cpp

```diff
@@ -18,21 +18,49 @@ void Attack::onInitialize() {
 }
 
 /// Get an update on the skill
+
 bt::Node::Status Attack::onUpdate() {
     if (! robot) return Status::Running;
 
     if (shot && !World::botHasBall(robot->id, true)) {
         return Status::Success;
     }
 
+
+    ballTarget = Field::get_their_goal_center();
+    genevaState = 3;
+
+    if (properties->hasInt("genevaState")) {
+        genevaState = properties->getInt("genevaState");
+    }
+
+    // Overwrite set genevaState if autoGeneva is true
+    if (properties->getBool("autoGeneva")) {
+        roboteam_msgs::GeometryFieldSize field = Field::get_field();
```

Comment:
these are some hardcoded genevastates... what does this 'autogeneva' do?

Reply:
Done